wremnants/syst_tools.py

```diff
@@ -498,8 +512,12 @@ def add_theory_hists(results, df, args, dataset_name, corr_helpers, qcdScaleByHe
         name = "ptVgen", underflow=False
     )
     #for hel analysis, ptVgen is part of axes/col
-    scale_axes = [*axes, axis_chargeVgen] if addhelicity else [*axes, axis_ptVgen, axis_chargeVgen]
-    scale_cols = [*cols, "chargeVgen"] if addhelicity else [*cols, "ptVgen", "chargeVgen"]
+    ## FIXME:
+    ## here should probably not force using the same ptVgen axis when addhelicity=True
```

Comment:
IS this comment still relevant?

Reply:
I am still not sure this is working correctly, although I think the part I commented was doing the wrong thing, so I ws keeping it with FIXME to remember to check asap.